### DISCUSSION TITLE
Templates

### DISCUSSION THREAD

**Discussion by BackOfClass** [nhaouari/obsidian-textgenerator-plugin#42] [Q&A] [answered]
Hi, I’m really interested in using AI assistance with my note taking in obsidian, and I saw your recent feature additions in the Templates. 

Could you please record a video showing this in action? 

Making a template, then using it to produce some content from it? I’m struggling to understand from the documentation alone.

Also, are all OpenAI models included?

**Comment by nhaouari** [comment, marked as answer]
Hi there,

Thanks for your interest in Text Generator plugin.  As soon as I finish testing templates and fixing bugs, I will publish videos explaining everything about them. 

Yes, all OpenAI models are included and with the frontmatter configuration you can also use other services such as huggingface.

Thanks again for your interest, and please let me know if you have any other questions.

**Reply by BackOfClass** [reply]
Awesome, I'm looking forward to it.

Do you have a rough estimate as to when this might happen?

Would love to get started using this.

Thanks.

**Reply by georgepor** [reply]
me too!

**Reply by bdo206** [reply]
Hello I too a really interested in this. Any videos explaining metadata best practices or templates?

**Reply by nhaouari** [reply]
Here a first demo video : [https://youtu.be/OergqWCdFKc](https://youtu.be/OergqWCdFKc)
